### PR TITLE
KD-8 Swagger-Ui-Documentation

### PR DESCRIPTION
**Pull request by ChuloWay**

### CODE REVIEW

**Review by tricelex — CHANGES_REQUESTED** [review]
There's something wrong with your folder structure.
Why is the user model outside the user directory? And why have nested user directory?
Please fix your folder structure, everything User should be in one directory.

File: src/model/user.schema.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    lastname: string;
+
+    @ApiProperty({
+        description: 'The Email Of The User'
+    })
+    email: string;
+
+    @ApiProperty({
+        description: 'The Phone Number Of the User'
+    })
```
Comment (tricelex) [inline src/model/user.schema.ts]: Please make your description uniform, check your cases

**Review by tricelex — COMMENTED** [review]
fix those then send again

**Comment by ChuloWay** [comment]
Okay

File: src/main.ts [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { SwaggerDocumentOptions } from './interface/swagger.interface';
+import { ExpressSwaggerCustomOptions} from './interface/swag.interface';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  app.useGlobalPipes( new ValidationPipe());
+  // app.useGlobalPipes(new ValidationPipe());
```
Comment (tricelex) [inline src/main.ts]: remove comment

Comment (ChuloWay) [reply src/main.ts]: Okay, just seeing it now

Comment (tricelex) [inline src/main.ts]: what is this nah?

**Review by tricelex — CHANGES_REQUESTED** [review]

**Review by tricelex — APPROVED** [review]

**Pull request merged**